CharacterCreation/03_Feats.txt

```diff
@@ -106,7 +106,7 @@ must be equal to or less than your strength.
 
 Empty the Mag: 
 Prerequisites:

```

Comment:
This being blank is going to parse as a section. Maybe we should add the word 'none' after the colon?

Reply:
sounds good